index.js

```diff
@@ -1003,8 +1023,11 @@ PersonaClient.prototype.getProfilesForGuids = function getProfilesForGuids(opts,
  * @private
  */
 PersonaClient.prototype._removeTokenFromCache = function (id, secret, callback) {
-    var cacheKey = "obtain_token:" + cryptojs.HmacSHA256(id, secret),
-        _this = this;
+    var cacheKey = this._formatCacheKey(
+        "obtain_token:" + crypto.createHash('md4').update(id).digest('base64')
```

Comment:
There are 3 places where crypto.createHash('md4').update(something).digest('base64') is done - can this be a function and just pass in the param?